### PR TITLE
Add Linky sensors : yesterday + months + years

### PR DESCRIPTION
**Pull request by Quentame**
**Description:**
Documentation for home-assistant/home-assistant#23726

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23726

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by Quentame** [comment]
Any news about this and home-assistant/home-assistant#23726 @klaasnicolaas ?

**Comment by klaasnicolaas** [comment]
Well there is a merge conflict that you need to solve first 😉

**Comment by Quentame** [comment]
Ok, haven't seen it, solving tonight.

It seems I haven't been notify of that :thinking:

**Comment by Quentame** [comment]
Both of my PRs were rebased and tested :wink: 
@klaasnicolaas

**Comment by klaasnicolaas** [comment]
Great but we still have to wait on the parent pr 😜

**Comment by Quentame** [comment]
Parent PR merged

File: source/_components/linky.markdown [hunk 1]
```diff
@@ -39,4 +44,42 @@ password:
   description: The Enedis account password.
   required: true
   type: string
+timeout:
+  description: Timeout to wait for the Enedis API connection.
+  required: false
+  type: integer
+  default: 10
+monitored_conditions:
```
Comment (frenck) [inline source/_components/linky.markdown]: This has been removed in the parent PR.

**Review by frenck — CHANGES_REQUESTED** [review]
@Quentame Could you please adjust the documentation with the last changes made in the parent PR? Thanks! 👍

**Comment by frenck** [comment]
PR adjusted. Removed the "tip" as well. While good at heart, when we started adding it everywhere in the docs, it might become annoying. Therefore, removed it.

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**